WalletWasabi.Fluent/Controls/TagsBox.axaml.cs

```diff
@@ -48,6 +48,9 @@ public class TagsBox : TemplatedControl
 		public static readonly StyledProperty<int> ItemCountLimitProperty =
 			AvaloniaProperty.Register<TagsBox, int>(nameof(ItemCountLimit));
 
+		public static readonly StyledProperty<int> MaxTextLengthProperty =
```

Comment:
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/TextBox.cs#L74

was going to suggest `MaxLength` as the property name, however this could be confusing so can stick with MaxTextLength.

Reply:
Yeah first I used the MaxLength name but I found it not that accurate.